src/platform/plugins/shared/discover/public/context_awareness/types.ts

```diff
@@ -331,6 +333,24 @@ export interface AdditionalCellAction {
   execute: (context: AdditionalCellActionContext) => void | Promise<void>;
 }
 
+/**
+ * Parameters passed to the open in new tab extension point action
+ */
+export interface OpenInNewTabExtPointAction {
```

Comment:
```suggestion
export interface OpenInNewTabParams {
```
Tiny nit: I'd go with a name that includes "params" for consistency.

Reply:
addressed!